### DISCUSSION TITLE
Hide [oh-my-zsh] Random theme '***' loaded

### DISCUSSION THREAD

**Discussion by InvisibleRasta** [ohmyzsh/ohmyzsh#10732] [Help] [answered]
Is there any way to hide this when starting a shell?

**Comment by carlosala** [comment, marked as answer]
Yes! You can set `ZSH_THEME_RANDOM_QUIET=true` before sourcing `oh-my-zsh.sh`.

**Reply by mcornella** [reply]
(This can be found in the [Settings wiki section](https://github.com/ohmyzsh/ohmyzsh/wiki/Settings)).
